include/EncodingSymbol.h

```diff
@@ -47,9 +47,10 @@ namespace LibFlute {
       EncodingSymbol(uint32_t id, uint32_t source_block_number, char* encoded_data, size_t data_len, FecScheme fec_scheme)
         : _id(id)
         , _source_block_number(source_block_number)
+        , _fec_scheme(fec_scheme)
```

Comment:
Isn't that initialised out of order now? Like not matching the order of the constructor arguments.